### DISCUSSION TITLE
Getting commits between builds based on tags returns no commits.

### DISCUSSION THREAD

**Discussion by sanalkr** [rfennell/AzurePipelines#1697] [Q&A]
We are trying to get commit messages between builds based on tags.  It can find a successful build with the given tag. When it's trying to get commits between builds, it cannot get any commits between those two builds. 

The Repo is in the Github private repository. There are two commits between these two builds. 
I tried to debug the task locally, and the statement below returns 0 commits. 

```
 globalCommits = await buildApi.getChangesBetweenBuilds(teamProject, lastGoodBuildId, buildId);
```

Here are the logs. 
![image](https://github.com/rfennell/AzurePipelines/assets/20180412/2b54861f-a6be-40e0-9211-9fc358e7bb3b)
![image](https://github.com/rfennell/AzurePipelines/assets/20180412/0b8d75ba-7ade-4198-a7e1-05e7a36d9240)

Any clue, why it's not able to get any commits? I am interested in getting commit messages.

**Comment by rfennell** [comment]
Sorry, I missed the notification this was logged, sorry for the delay.....

What do you mean by Tags?

I ask as a common confusion is that people expect Tags to be Git Tags, when the task only support Azure DevOps pipeline tags?

The log appears to show no tags can be found on the scan pipeline runs
